src/main/java/frc/trigon/robot/commands/LEDAutoSetupCommand.java

```diff
@@ -62,23 +52,15 @@ private Command getUpdateAutoStartPoseCommand() {
         ).ignoringDisable(true);
     }
 
-    private Color getLeftStripColor(double difference, double tolerance) {
+    private edu.wpi.first.wpilibj.util.Color getStripDesiredColor(double difference, double tolerance) {
```

Comment:
This should just be `set` not `get`

Reply:
No, it doesn't set the color. It get's the color.